.storybook/sass.ts

```diff
@@ -0,0 +1,389 @@
+const sass_values = {
```

Comment:
I worry we'll forget to update this file as we evolve our mixins. Is there a path forward that could move this documentation directly adjacent to the code of interest?

Reply:
Do you mean just moving the location or eventually having it autogenerate based on the sass file?

Reply:
Kinda tricky as the `styles` directory contains auto generated files that we ignore. I wouldn't want to to include this file with the built package. Perhaps it can instead be put into the `docs` directory of the `pharos` package?

Reply:
What I'm suggesting is using something like [SassDoc](http://sassdoc.com/) directly into the SASS mixin code and extracting it from there.

This can certainly be a future endeavor, but I think that'd be the ideal and would parallel the way we document the components' APIs.

Reply:
Ah I see. Yes something like that would be ideal.

Reply:
I'll file a new issue for that

Reply:
That's now #127